feature_engine/discretisation/__init__.py

```diff
@@ -4,6 +4,7 @@
 """
 
 from .arbitrary import ArbitraryDiscretiser
+from .base_discretiser import BaseDiscretiser
```

Comment:
We do not add base classes to the init. In the init we only add the transformers that are meant to be used by external users.

The base classes are meant for us, developers :p

Can you please remove it?

Reply:
Ok, good to know. I was unsure. PyCharm suggested it. I assumed the IDE knew better than me. I'm glad to know PyCharm cannot replace me yet ;)